lib/scanner.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const objectScan = require('object-scan');
+
+module.exports = (haystack, needles) => {
```

Comment:
Please give the function the name `findInTree`, for better stack trace readability.

Reply:
done